src/ethereum_test_tools/common/types.py

```diff
@@ -1166,6 +1181,7 @@ class Header:
     withdrawals_root: Optional[bytes | Removable] = None
     data_gas_used: Optional[int | Removable] = None
     excess_data_gas: Optional[int | Removable] = None
+    parent_beacon_block_root: Optional[bytes | Removable] = None
```

Comment:
After #217, this definition would change to:
```
    parent_beacon_block_root: Optional[Hash] = header_field(
        default=None,
        header_field=HeaderField(
            fork_requirement_check="header_beacon_root_required",
            environment_source="parent_beacon_block_root",
            cast_type=Hash,
        ),
        json_encoder=JSONEncoder.Field(name="beaconRoot"),
    )
```

We also might need to add it to `FixtureExecutionPayload`, but we need to rebase first.